### PR TITLE
[CDE-1052] RequireJS and ${solution:mypath/myfile} in html - image not displayed

### PR DESCRIPTION
**Pull request by moraesvc**
@ssamora 
@smmribeiro 
@bcostahitachivantara

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 42s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **395**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/cde-plugin/job/PR-991/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 57s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **396**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/cde-plugin/job/PR-991/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 26s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **396**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/cde-plugin/job/PR-991/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 7s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **396**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/cde-plugin/job/PR-991/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 4m 38s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **396**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/cde-plugin/job/PR-991/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: core/impl/src/main/java/pt/webdetails/cdf/dd/model/inst/writer/cdfrunjs/dashboard/amd/PentahoCdfRunJsDashboardWriteContext.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
 public class PentahoCdfRunJsDashboardWriteContext extends CdfRunJsDashboardWriteContext {
   private static final String SIMPLE_TOKEN = "\\$\\{(\\w+)\\}";
   private static final String RESOURCE_TOKEN = "\\$\\{(\\w+):((/?)(.+?)(/?))\\}";
+  final Pattern SimplePattern = Pattern.compile( SIMPLE_TOKEN );
```
Comment (afrjorge) [inline core/impl/src/main/java/pt/webdetails/cdf/dd/model/inst/writer/cdfrunjs/dashboard/amd/PentahoCdfRunJsDashboardWriteContext.java:29]: `simplePattern`

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 51s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **396**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/cde-plugin/job/PR-991/6/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 33s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **396**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/cde-plugin/job/PR-991/7/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 48s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **396**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/cde-plugin/job/PR-991/8/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 4m 16s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **396**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/cde-plugin/job/PR-991/9/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 4m 11s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **396**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/cde-plugin/job/PR-991/10/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by ssamora — APPROVED** [review]

**Pull request merged**